build/debian-hyperkube-base/Makefile

```diff
@@ -24,7 +24,7 @@ ARCH?=amd64
 ALL_ARCH = amd64 arm arm64 ppc64le s390x
```

Comment:
the TAG needs updating, right?

Reply:
Yes ,and I will create a new PR to add mips64le for debian-hyperkube-base, this PR will just  update the tag of BASEIMAGE.

Reply:
typically, changes to this file that modify the resulting image also update TAG in the same commit

Reply:
OK, I will update the TAG in this PR.

Reply:
There is a problemthat even if this TAG update to v1.0.0, I need to create a PR fo release-1.16 to build a mips64le v1.0.0 debian-base image. Because in branch release-1.17 and master, the debian-base image is already use the TAG v2.0.0